math/moserdebruijnsequence/sequence.go

```diff
@@ -0,0 +1,37 @@
+// The Moser-de Bruijn sequence is the sequence obtained by
+// adding up the distinct powers of the number 4 (For example 1, 4, 16, 64, etc).
```

Comment:
You should probably add the Wikipedia URL:
https://en.wikipedia.org/wiki/Moser%E2%80%93de_Bruijn_sequence